### PR TITLE
[Merged by Bors] - feat(RingTheory/Valuation): add one lemma `isEquiv_iff_val_lt_val`

### PR DESCRIPTION
**Pull request by jjdishere**
Add a single lemma `Valuation.isEquiv_iff_val_lt_val`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

thanks!

File: Mathlib/RingTheory/Valuation/Basic.lean [hunk 1]
```diff
@@ -412,6 +412,12 @@ theorem isEquiv_of_map_strictMono [LinearOrderedCommMonoidWithZero Γ₀]
     (H : StrictMono f) : IsEquiv (v.map f H.monotone) v := fun _x _y =>
   ⟨H.le_iff_le.mp, fun h => H.monotone h⟩
 
+theorem isEquiv_iff_val_lt_val [LinearOrderedCommGroupWithZero Γ₀]
+    [LinearOrderedCommGroupWithZero Γ'₀] (v : Valuation K Γ₀) (v' : Valuation K Γ'₀) :
+    v.IsEquiv v' ↔ ∀ {x y : K}, v x < v y ↔ v' x < v' y :=
+  ⟨fun h x y => by simpa only [not_le] using (h y x).not,
+    fun h _ _ => by simpa only [not_lt] using h.not⟩
```
Comment (urkud) [inline Mathlib/RingTheory/Valuation/Basic.lean:419]: Optional suggestion (not tested):
```suggestion
    v.IsEquiv v' ↔ ∀ {x y : K}, v x < v y ↔ v' x < v' y := by
  simp only [IsEquiv, le_iff_le_iff_lt_iff_lt]
```

File: Mathlib/RingTheory/Valuation/Basic.lean [hunk 2]
```diff
@@ -412,6 +412,12 @@ theorem isEquiv_of_map_strictMono [LinearOrderedCommMonoidWithZero Γ₀]
     (H : StrictMono f) : IsEquiv (v.map f H.monotone) v := fun _x _y =>
   ⟨H.le_iff_le.mp, fun h => H.monotone h⟩
 
+theorem isEquiv_iff_val_lt_val [LinearOrderedCommGroupWithZero Γ₀]
+    [LinearOrderedCommGroupWithZero Γ'₀] (v : Valuation K Γ₀) (v' : Valuation K Γ'₀) :
```
Comment (urkud) [inline Mathlib/RingTheory/Valuation/Basic.lean:416]:
```suggestion
    [LinearOrderedCommGroupWithZero Γ'₀] {v : Valuation K Γ₀} {v' : Valuation K Γ'₀} :
```

File: Mathlib/RingTheory/Valuation/Basic.lean [hunk 3]
```diff
@@ -412,6 +412,12 @@ theorem isEquiv_of_map_strictMono [LinearOrderedCommMonoidWithZero Γ₀]
...
+    v.IsEquiv v' ↔ ∀ {x y : K}, v x < v y ↔ v' x < v' y :=
+  ⟨fun h x y => by simpa only [not_le] using (h y x).not,
+    fun h _ _ => by simpa only [not_lt] using h.not⟩
+
```
Comment (urkud) [inline Mathlib/RingTheory/Valuation/Basic.lean:420]: alias ⟨IsEquiv.lt_iff_lt, _⟩ := isEquiv_iff_val_lt_val

**Comment by urkud** [comment]
I left some suggestions, otherwise LGTM. Thanks!
bors d+

File: Mathlib/RingTheory/Valuation/Basic.lean (hunk 2 above)
Comment (jjdishere) [reply Mathlib/RingTheory/Valuation/Basic.lean:416]: I use `(v v')` here to be aligned with `isEquiv_of_val_le_one` and other lemmas. Do you think changing all `(v v')` into `{v v'}` is a good idea? I will make the change if you think this is a good idea.

File: Mathlib/RingTheory/Valuation/Basic.lean (hunk 3 above)
Comment (jjdishere) [reply Mathlib/RingTheory/Valuation/Basic.lean:422]: Added this alias. Thanks!

File: Mathlib/RingTheory/Valuation/Basic.lean (hunk 2 above)
Comment (jjdishere) [reply Mathlib/RingTheory/Valuation/Basic.lean]: I'll open a new PR to change parenthesis in similar lemmas.

**Pull request closed without merging**